common/lib/xmodule/xmodule/tabs.py

```diff
@@ -390,7 +390,7 @@ def initialize_default(course):
         course.tabs.extend([
             CourseTab.load('textbooks'),
             discussion_tab,
-            CourseTab.load('wiki'),
+            CourseTab.load('news'),
```

Comment:
I still think that it's better to not replace the old tabs with new ones, but create new tabs separately, irrespectively of old tabs. But it's OK)

Reply:
TBD